lib/portlayer/exec/container.go

```diff
@@ -220,6 +230,10 @@ func (c *ContainerInfo) State() State {
 	return c.state
 }
 
+func (c *Container) String() string {
```

Comment:
Since we already have `containerBase.String()` and `ContainerInfo.String()` (in container.go), do we really need this method?

Reply:
Not really, but one day someone could change one of those to output something else and I didn't want to get affected by it.